check_replication_queue.py

```diff
@@ -0,0 +1,32 @@
+from client import get_client
+import sys
+import waiting
+import click
+
+
+SNUBA_ADMIN_HOST = "127.0.0.1"
+CLICKHOUSE_CLIENT_PORT = "9000"
+
+def _is_replication_queue_empty(client):
+    return len(client.execute("SELECT * FROM system.replication_queue LIMIT 1;")) == 0
+
+
+@click.command()
+@click.option("--host", default=SNUBA_ADMIN_HOST)
+def check_replication_queue(host: str):
+    client = get_client(host, CLICKHOUSE_CLIENT_PORT)
+    try:
+        waiting.wait(lambda: _is_replication_queue_empty(client), sleep_seconds=60, timeout_seconds=300)
+    except waiting.TimeoutExpired as e:
```

Comment:
What kind of errors get thrown if there are problems connecting to the clickhouse server? does `waiting` re-raise that error?

Reply:
My understanding is that waiting does not disconnect the client. Throughout the time threshold, the client will remain connected, so if there is a problem connecting to clickhouse, it would throw a network error at the start instead of during `waiting`